paymentsheet/src/main/java/com/stripe/android/paymentsheet/Element.kt

```diff
@@ -110,4 +116,44 @@ internal sealed class SectionFieldElement {
         override val identifier: IdentifierSpec,
         override val controller: DropdownFieldController,
     ) : SectionFieldElement()
+
+    internal class AddressElement @VisibleForTesting constructor(
+        override val identifier: IdentifierSpec,
+        private val addressFieldRepository: AddressFieldElementRepository,
+        countryDropdownFieldController: DropdownFieldController
```

Comment:
If you change the default constructor value to
```
countryDropdownFieldController: DropdownFieldController =DropdownFieldController(CountryConfig(emptySet()))
```
then the constructor in 126 can be removed